api-gateway/src/test/java/com/petclinic/bffapigateway/domainclientlayer/AuthServiceClientIntegrationTest.java

```diff
@@ -27,8 +35,13 @@
  * Date: 2021-10-15
  * Ticket: feat(APIG-CPC-354)
  */
+@RequiredArgsConstructor
 public class AuthServiceClientIntegrationTest {
 
+    private final SecurityConst securityConst = new SecurityConst(60
+            ,"MIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDTnkqggdMFu5O58pB8U0f8D4pab_7j5T8jNZfEep23DbvoMCjR6X1cFe4qCsvaY4aDF6d6Vn3TVY2INHMuyOTXqe5vhBWiRgaI3TIPkGjgHZ1f6Up_ZPQFJCGTo2b3OiXBq3LTK7PXvMj2EIQPJrHuX099ACDvO"
+            ,"Bearer");
+
     private AuthServiceClient authServiceClient;
     private MockWebServer server;
     private ObjectMapper objectMapper;
```

Comment:
This will make tests pass for things like registration even though they don't work in reality. It's not false, but it doesn't match how it would get used in reality

Reply:
It actually does work in the repo ! Just not from the controller.